### DISCUSSION TITLE
blocked

### DISCUSSION THREAD

**Discussion by gooseisgoose** [3kh0/ext-remover#739] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm confused what if bookmarklets are blocked.

**Comment by 3kh0** [comment, marked as answer]
You can check if bookmarklets are blocked by going to `chrome://policy`

**Reply by gooseisgoose** [reply]
ok thanks

**Reply by gooseisgoose** [reply]
so is there no way to enable them

**Reply by 3kh0** [reply]
> so is there no way to enable them

If they have been disabled by your admin, you cannot enable them.

**Reply by gooseisgoose** [reply]
ok
